site/topics/genetic-programming/genetic-programming.rst

```diff
@@ -154,10 +154,105 @@ Representation
 Language
 --------
 
+* The set of available operators and operands use for creating and modifying the trees is called the *language*
+* The operators and operands are something that can be adjusted as needed
+
+* With the breast cancer example, the language could be a collection of arithmatic and boolean operators and operands
+
+    * Operators
+
+        * Binary Operators
+
+            * :math:`+`
+            * :math:`-`
+            * :math:`\times`
+            * :math:`/`
+            * :math:`<`
+            * :math:`>`
+            * :math:`==`
+            * and
+            * or
+
+        * Unary Operators
+
+            * :math:`sin`
+            * :math:`cos`
+            * :math:`e`
```

Comment:
_e_ is kinda more binary? _base_ and _exponent_?

Reply:
Not really? _e_ is a number, so I can do anything to it, like add it to other numbers, or run it through the Ackerman function.

Reply:
But then it would be compounded (for lack of a better term) with another operator to form more complex expressions.

Reply:
That is my point. _e_ isn't a function, but it is listed as such. Exponentation is a binary operator.

Reply:
I see what you're saying. The e in this scenario is being treated as the exponential function. So, there is no exponent operator in the language, only the ability to raise e to the power of some value.